### PR TITLE
WSL Non-Root User Check

### LINKED ISSUE #165: WSL User Account Creation
Students sometimes end up skipping the step in the WSL setup tutorial where they create a user account. They should be prompted when they open the Ubuntu terminal as directed by the setup tutorial, but it doesn't directly mention this process.

Calling this out explicitly might help. We can also suggest best practices, e.g. something like what's show [here](https://learn.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-username-and-password). I wouldn't suggest linking to that page, though, since it has a bunch of irrelevant stuff. Maybe these points?
- choose a user name that is all lowercase characters (could be your uniqname)
- while typing your password, the characters aren't shown. that's fine
- you can just hit enter to skip the other things (e.g. "room number" lol)

Also add a "pitfall" that has them check they aren't root. If they are, we could just suggest uninstalling and reinstalling WSL (if they have just installed it). Or, if students already had a previous installation, we might be able to find a resource to tell them how to create a user account, give it privileges, and make it the default user (this process is a bit complicated).

### PR DESCRIPTION
**Pull request by ohjuny**
Fixes #165 

Note: the uninstall link is a random article I found online because I couldn't find any uninstall instructions from our end. Should we add "official" uninstall instructions to this page, or is this fine?

### CODE REVIEW

**Review by awdeorio — COMMENTED** [review]
I pushed a couple of suggestions, take a look and see what you think.

I really like your idea.  Let's do it.
> Should we add "official" uninstall instructions to this page?

File: docs/setup_wsl.md
Comment (ohjuny) [inline docs/setup_wsl.md:92]: Is the mixed casing in "SOMEthiNG IS WRONG" intentional?

File: Gemfile.lock
Comment (ohjuny) [inline Gemfile.lock:262]: Unsure what this does but assuming this is fine.

**Review by ohjuny — COMMENTED** [review]
As for creating official uninstall instructions, I'm hesitant based on my experience writing #99  - unsure if it's worth the maintenance burden. Also, I don't have a Windows machine, so I feel like it would just be copying instructions (and screenshots) from another online guide.

In office hours, I've been pointing students to [this guide](https://www.makeuseof.com/uninstall-wsl-windows/). They seemed to have no trouble following it, and it seemed to work fine. Thoughts on just using this link for now?

File: docs/setup_wsl.md
Comment (awdeorio) [reply docs/setup_wsl.md:92]: Yes, but please go ahead and change it to your preference.  No big deal.

**Comment by awdeorio** [comment]
> As for creating official uninstall instructions, I'm hesitant based on my experience writing https://github.com/eecs280staff/tutorials/pull/99 - unsure if it's worth the maintenance burden. Also, I don't have a Windows machine, so I feel like it would just be copying instructions (and screenshots) from another online guide.

> In office hours, I've been pointing students to [this guide](https://www.makeuseof.com/uninstall-wsl-windows/). They seemed to have no trouble following it, and it seemed to work fine. Thoughts on just using this link for now?

Great suggestion!  Your idea is better.

File: docs/setup_wsl.md
Comment (ohjuny) [reply docs/setup_wsl.md:92]: Nice, lol

**Comment by ohjuny** [comment]
Ready for review+merge!

File: docs/.DS_Store
Comment (awdeorio) [inline docs/.DS_Store:1]: I suspect you were using `git add .` or similar!  This looks like an accidental file add.

**Review by awdeorio — APPROVED** [review]
LGTM after removing all the stray `.DS_Store` files.  I'll approve so you can merge without me in the loop.

File: docs/.DS_Store
Comment (ohjuny) [reply docs/.DS_Store:1]: Woops my bad!

**Comment by ohjuny** [comment]
> LGTM after removing all the stray `.DS_Store` files. I'll approve so you can merge without me in the loop.

Apparently I don't have merge permissions in the tutorials repo :(

**Comment by jamesjuett** [comment]
Whoops, I had this on my to-do list for today but missed the boat on reviewing it. Overall, LGTM, but I'll follow up in a separate issue about a potential improvement on the uninstall instructions.

**Pull request merged**
